src/scss/_anchor-box.scss

```diff
@@ -0,0 +1,37 @@
+/// @access private
+/// @output Styling for anchor 'controls'
+
+@mixin _oFormsAnchorBox {
+	.o-forms--anchor {
```

Comment:
I don't think it makes sense to call this just `anchor`, but a good name is hard. ` o-forms--anchor-radio-box`, ` o-forms--pseudo-radio-box`, ` o-forms--fake-radio-box`?